experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -72,27 +72,27 @@ export interface Counter {
   /**
    * Increment value of counter by the input. Inputs may not be negative.
    */
-  add(value: number, attributes?: Attributes, context?: Context): void;
+  add(value: number, attributes?: SpanAttributes, context?: Context): void;
```

Comment:
`SpanAttributes` seems to be strange here. I assume this is used because `Attributes` is a 1.1 type.
What about adding a `MetricsAttributes`/`MeterAttributes` in metrics API package until it gets merged into API? This merge must result in a API 2.0 then as it would be hard to have a metrics SDK based on 1.0 API.

Reply:
Yes. If we use a 1.1 type, then the module cannot be used with any `<1.1` API. There is unfortunately no way to specify a different type for a different installed API.